<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>921</width>
      <height>721</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="holls_per_round">
    <property name="geometry">
     <rect>
      <x>939</x>
      <y>53</y>
      <width>75</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimum">
     <double>30.000000000000000</double>
    </property>
    <property name="maximum">
     <double>200.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>30</y>
      <width>121</width>
      <height>74</height>
     </rect>
    </property>
    <property name="text">
     <string>holls per round</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_decrease">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>100</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zoom_increase">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>100</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QListWidget" name="test_files_list">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>150</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rebuild_btn">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>290</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>man rebuild</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="paint_angles">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>240</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>paint angles</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
